grafana/dashboards/pod.json

```diff
@@ -15,8 +15,7 @@
   "editable": true,
   "gnetId": null,
   "graphTooltip": 1,
-  "id": null,
-  "iteration": 1531434418636,
+  "iteration": 1552075566256,
```

Comment:
Can I trouble you to manually revert the change to the "id" and "iteration" fields here?

Reply:
sure will do that